### ISSUE TITLE
Automation Script to connect remote server and install devtron after creation of cluster from Devtron's app

### ISSUE THREAD

**Issue by deepak-devtron** [devtron-labs/devtron#1643]
Here I am doing POC for resolving Automation problem where I have to create a KOPs cluster after connecting to remote bastion from Devtron's custom App >>>Deploy Devtron in same cluster>>>Run Automation in same cluster>>> Delete Cluster

This will help us to run our automation to run dynamically in any newly created cluster.

**Comment by deepak-devtron** [comment]
Already invested 3 days in this sprint ,as of now I am blocked on this so closing it and create a new task for the same in next sprint for the specific work which is remaining.
